app/components/chronos-submission-panel/template.hbs

```diff
@@ -0,0 +1,23 @@
+{{#cp-panel}}
+    {{#cp-panel-toggle}}
+        <p>Submit to journals on Chronos</p>
+    {{/cp-panel-toggle}}
+
+    {{#cp-panel-body}}
+        <div>
+            {{#power-select search=(perform _fetchInitialJournals) optionsComponent=(component 'lazy-options' loadMore=(action "getMoreJournals") isLoading=isLoading canLoadMore=canLoadMore) options=journals placeholder="Select a journal" selected=selectedJournal onfocus=(action "getInitialJournals" "") oninput=(action "getInitialJournals") onchange=(action "journalSelected") as |journal|}}
+                <strong> {{journal.title}}</strong>
+            {{/power-select}}
+        </div>
+        {{#if selectedJournal}}
+            <div style="padding-top: 5px">
```

Comment:
add a class

Reply:
Adding a new CSS class for the sole purpose of adding a 5px top padding doesn't sound ideal. I've checked elsewhere in the codebase and in-line styling seems to be used for these kind of paddings.

Reply:
convincing enough :+1: